source/_components/switch.rainmachine.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: page
+title: "RainMachine Switch"
+description: "Instructions on how to use RainMachine units with Home Assistant."
+date: 2017-08-04 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: rainmachine.png
+ha_category: Switch
+ha_iot_class: "Cloud Polling"
+ha_release: 0.50.3
+---
+
+The `rainmachine` switch platform allows you to control programs and zones within
+a [RainMachine smart Wi-Fi sprinkler controller](http://www.rainmachine.com/).
+
+## {% linkable_title Configuring the Platform %}
+
+The platform allows for both local (i.e., directly across the LAN) and remote
```

Comment:
I think it should be "either local or or remote since looking the config examples the switch is configured for one or the other.